dev/com.ibm.ws.logging/src/com/ibm/ws/logging/collector/CollectorJsonHelpers.java

```diff
@@ -205,6 +219,66 @@ private static void addUnchangingFields1_1(StringBuilder sb, String hostName, St
 
     }
 
+    private static void addUnchangingFieldsJSON_Message(StringBuilder sb, String hostName, String wlpUserDir, String serverName, boolean isMessageEvent) {
+        LogTraceData logtracedata = new LogTraceData();
```

Comment:
once you make the key-getter methods static you shouldn't need to instantiate LogTraceData to get its keys (same comment applies below to all cases where you are instantiating *Data classes just to get key names)